spacy/ml/tb_framework.pyx

```diff
@@ -113,9 +127,7 @@ def init(
     inferred_nO = _infer_nO(Y)
     if inferred_nO is not None:
         current_nO = model.maybe_get_dim("nO")
-        if current_nO is None:
-            model.set_dim("nO", inferred_nO)
-        elif current_nO != inferred_nO:
+        if current_nO is None or current_nO != inferred_nO:
```

Comment:
Why was `model.set_dim("nO", inferred_nO)` removed here?

Reply:
It's set `resize_output`.